docs/edge-authorizers/deployment-and-operation.mdx

```diff
@@ -75,9 +72,99 @@ docker run \
 -p 8282:8282 \
 -p 8383:8383 \
 -p 8484:8484 \
--v $PWD/certs:/certs:rw \
--v $PWD/cfg:/cfg:ro \
--v $PWD/eds:/db:rw \
+-p 9292:9292 \
+-v $HOME/.config/aserto/sidecar/certs:/certs:rw \
+-v $HOME/.config/aserto/sidecar/cfg:/cfg:ro \
+-v $HOME/.config/aserto/sidecar/eds:/db:rw \
 ghcr.io/aserto-dev/topaz:latest run \
---config-file /cfg/config.yaml
+--config-file /cfg/<POLICY_INSTANCE_NAME>.yaml
+```
+
+## Deploy as a Sidecar
+
+Deploying topaz as a sidecar next to your applications is the most efficient way to use the authorizer as it minimizes the authorization latency.
```

Comment:
Topaz

Reply:
this is the only place we use Topaz instead of Edge Authorizer, maybe we use it more?

Reply:
Hmm. I believe we mention Topaz in a few places which was intentional.

Reply:
Thinking about this more, I don't think we need to mention Topaz here, I changed both this and the next sentence to use "Edge Authorizer" vs "Topaz"